decidim-admin/app/views/decidim/admin/bulk_actions/_dropdown.html.erb

```diff
@@ -0,0 +1,35 @@
+<div id="js-bulk-actions-wrapper">
+  <button
+    id="js-bulk-actions-button"
+    class="button button--nomargin tiny success light dropdown"
+    type="button"
+    data-toggle="js-bulk-actions-dropdown"
+    aria-controls="js-bulk-actions-dropdown"
+    data-is-focus="false"
+    data-yeti-box="js-bulk-actions-dropdown"
+    aria-haspopup="true"
+    aria-expanded="false">
```

Comment:
Are you sure you need so many attributes for this? See https://github.com/decidim/decidim/blob/master/decidim-admin/app/views/decidim/admin/exports/_dropdown.html.erb which is used for the "Export" dropdown in proposals, for example, it doesn't need so many attributes. I think some of these are automatically added by foundation

Reply:
I will leave only the indispensable